### PR TITLE
43 search results

### PR DESCRIPTION
**Pull request by jonatan-lledo-netcentric**
Fix #43

Test URLs:
- Before: https://main--vg-roadchoice-com--hlxsites.hlx.page/
- After: https://43-search-results--vg-roadchoice-com--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/filters/filters.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  const currentUrl = new URL(window.location.href);
+  const urlParams = new URLSearchParams(currentUrl.search);
+
+  cats.forEach((cat) => {
+    const [category, amount] = cat;
+    urlParams.set('cat', category);
+    const filterUrl = `${currentUrl.pathname}?${urlParams.toString()}`;
+    const item = createElement('li', { classes: 'item' });
+    const link = createElement('a', {
+      classes: 'categories-link',
```
Comment (SantiagoHomps-NC) [inline blocks/filters/filters.js:49]: I see that when doing this when we select a category, we go to a new page and leave behind all the others.. could we just hide the products instead? maybe just using a class and hiding the category not selected? WDYT?

File: blocks/blog-results/blog-results.js [hunk 2]
```diff
@@ -339,6 +339,7 @@ const buildSidebar = (articles, titleContent) => {
 
 export default async function decorate(block) {
   const blockConfig = readBlockConfig(block);
+  console.log(blockConfig);
```
[the remark this answers is not in the document]
Comment (jonatan-lledo-netcentric) [reply blocks/blog-results/blog-results.js]: removed

File: blocks/filters/filters.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      props: { href: filterUrl },
+      textContent: `${category} (${amount})`,
```
[the remark this answers is not in the document]
Comment (jonatan-lledo-netcentric) [reply blocks/filters/filters.js]: done

File: blocks/results-list/results-list.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+/* Product card styling */
+.product {
+  padding: 0 10px 15px;
+  margin: 0 0 10px;
+  width: 100%;
+  min-height: 300px;
+  display: inline-flex;
+  flex-direction: column;
+}
```
[the remark this answers is not in the document]
Comment (jonatan-lledo-netcentric) [reply blocks/results-list/results-list.css:17]: centered

File: blocks/filters/filters.js (hunk 1 above)
Comment (jonatan-lledo-netcentric) [reply blocks/filters/filters.js:49]: using back in the browser we recover the previous search, so I prefer to leave it as this. but we can ask them if this is ok

**Review by SantiagoHomps-NC — APPROVED** [review]

**Pull request merged**